{"name": "test_valid_login[credentials0]", "status": "failed", "statusDetails": {"message": "AssertionError: assert False\n +  where False = <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>>()\n +    where <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>> = <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>.is_dashboard_text_visible", "trace": "driver_manager = <src.utils.driver_manager.DriverManager object at 0x0000028EA98609D0>, credentials = {'password': 'Password123', 'username': 'student'}\n\n>   ???\nE   assert False\nE    +  where False = <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>>()\nE    +    where <bound method LoginPage.is_dashboard_text_visible of <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>> = <src.pages.login.login_page.LoginPage object at 0x0000028EA98742D0>.is_dashboard_text_visible\n\nd:\\Selenium Project\\web_automation_selenium - Copy\\src\\tests\\login\\test_login.py:23: AssertionError"}, "parameters": [{"name": "credentials", "value": "{'username': 'student', 'password': 'Password123'}"}], "start": 1730972797515, "stop": 1730972800595, "uuid": "2d350241-be5c-47df-abcf-0ebca49a0d28", "historyId": "9508d0765b0b1f8bcccda8a6b0c295ef", "testCaseId": "2e11a866fb8562c8d371512e0099dafd", "fullName": "src.tests.login.test_login#test_valid_login", "labels": [{"name": "parentSuite", "value": "src.tests.login"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "DESKTOP-1ML9BAE"}, {"name": "thread", "value": "10012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.login.test_login"}]}